test/completeness/conftest.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+
+from pytest import fixture
+from pandas import read_csv
+import numpy as np
+import skdh
+
+@fixture(scope="module")
+def completeness_sub_data():
+    cwd = str(Path.cwd())
+    if cwd.split('/')[-1] == "completeness":
+        subject_folder = cwd + '/data/'
```

Comment:
`cwd` is not goint to work since these tests can be run from multiple places. Please see other tests that load data (ie gait) for how to handle this properly